CHANGELOG.md

```diff
@@ -4,13 +4,26 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## 0.7.0
+
+The main change in this release is that it makes use of a new `scale-encode` crate and updated `scale-decode` crate for the SCALE encoding and decoding of `Value`s.
```

Comment:
Value's looks a bit funny with syntax highlighting for `Value`s, isn't `Value's` better? :)

```suggestion
The main change in this release is that it makes use of a new `scale-encode` crate and updated `scale-decode` crate for the SCALE encoding and decoding of `Value's`
```

Reply:
Yeah maybe it was pedantic of me; `Value` is a type from the code but the "s" isn't, and "'s" isn't really correct grammar. Maybe I should have just said `Values`, but yeah!

Reply:
I just removed the highlighting instead!